.eslintrc.json

```diff
@@ -23,6 +23,9 @@
       "build/",
       "App.js",
       "index.js",
-      "App.test.js"
+      "App.test.js",
+      "Todo.js",
+      "TodoForm.js",
+      "TodoList.js"
     ]
```

Comment:
[ ] Please make sure you do not edit the linters, the correct use of linters helps us to improve the way we write code.

### Reference

![image](https://github.com/ahmed025566/To-Do-App/assets/78930134/0b18ac1d-b74b-4a45-984c-2dc6773e5c26)